From 5461dd2bc08dc1036b655c8aa9f8acf306b092fd Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Thu, 18 Feb 2016 13:21:59 +0100
Subject: [PATCH] ip-address.8.in: fix BNF syntax error

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1231898
Upstream Status: iproute2.git commit 715296b85a0ec

commit 715296b85a0ecece7cdc60d42b026d66ac21a55e
Author: Phil Sutter <phil@nwl.cc>
Date:   Tue Aug 18 18:11:12 2015 +0200

    ip-address.8.in: fix BNF syntax error

    The previous man page fixup introduced a syntax error due to missing
    opening bracket, which might crash some humanoid BNF parsers.

    Fixes: 4e972d5 ("ip-address: fix and extend documentation")
    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/ip-address.8.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man8/ip-address.8.in b/man/man8/ip-address.8.in
index 2ea3e67..f1efc43 100644
--- a/man/man8/ip-address.8.in
+++ b/man/man8/ip-address.8.in
@@ -60,7 +60,7 @@ ip-address \- protocol address management
 .IR FLAG " := "
 .RB "[ " permanent " | " dynamic " | " secondary " | " primary " | \
 [ - ] " tentative " | [ - ] " deprecated " | [ - ] " dadfailed " | "\
-temporary " ] " CONFFLAG-LIST " ]"
+temporary " ] [ " CONFFLAG-LIST " ]"
 
 .ti -8
 .IR CONFFLAG-LIST " := [ "  CONFFLAG-LIST " ] " CONFFLAG
-- 
1.8.3.1

